app/views/GermlineView/components/Board/components/ApiPaginatedTable/components/CheckboxCell/index.scss

```diff
@@ -0,0 +1,3 @@
+.checkbox-cell {
+  padding: 0 !important;
```

Comment:
are there absolutely no other way than to use !important?

Reply:
It didn't like it without in testing, but now it seems fine ?_?
Anyways, I removed it